.gitignore

```diff
@@ -41,3 +41,5 @@
 
 # Makefile
 /Makefile
+
+/.cache/
```

Comment:
Please use your local setting

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer
https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file